### DISCUSSION TITLE
Copilot only suggest one line in VSCode

### DISCUSSION THREAD

**Discussion by SoyDiego** [community/community#58520] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I read about this topic https://github.com/orgs/community/discussions/13303 and say it's solved but I continue with the same problem.
Copilot on Visual Studio works perfectly and recommends multiple line, but in my VSCode, only suggest one line all time.

![image](https://github.com/community/community/assets/19764334/1b7859f8-91e5-43eb-9d0a-5b8e75574cf5)

And also sometimes suggest lines but like comments (starting with `//`).

Someone with the same problems?

**Comment by crysis992** [comment]
Same issue for me. Started the copilot trial today and it's not showing multi line suggestions, it's not able to generate or suggest something from a comment. I'm working only in react/typescript projects, didnt test in any other language.

And if it's showing a suggestion it's some garbage like the screenshot below:
Suggesting just a comment/link to stackoverflow

![Screenshot 2023-06-20 175921](https://github.com/community/community/assets/5308174/e8925092-ff05-400b-a213-ad2d8f4660d7)

Pressing ctrl +enter opens the copilot window and that gives some useful completion, but thats like an extra step i have to do every time.

The copilot output has no errors, I'm signed in, extension and vscode is on the latest version
![Screenshot 2023-06-20 180721](https://github.com/community/community/assets/5308174/21a17a15-f03e-4d16-afc0-32d308c4e9aa)

**Comment by SoyDiego** [comment]
Is there no solution to this problem?

**Reply by crysis992** [reply]
I've switched to the nightly build of the extension and uninstalled the stable one (Github Copilot Nightly) and it's working since then

**Reply by SoyDiego** [reply]
Thanks @crysis992 I will try to do it. I'm not in the trial, my company paid for copilot and it's weird haha. I will confirm in a while if I can fix it with your recommendation. Thanks!

**Reply by SoyDiego** [reply]
I tried with the Nightly version and it's the same :(

![image](https://github.com/community/community/assets/19764334/321d9bd0-373d-4085-ac0f-039328f8bf17)

**Comment by Samielsolh** [comment]
Same problem here...
